fairing/runtime_config.py

```diff
@@ -23,6 +23,12 @@ def set_deployer(self, name, **kwargs):
     def get_deployer(self, **kwargs):
         return self._deployer
 
+    def get_model(self):
```

Comment:
These aren't necessary anymore. What crash were you seeing?

Reply:
hmm I think it confuses the name. because we have config module then inside the config you instantiate the Config() class and RuntimeConfig to `config`. and when call fairing.config I think the second one is called.
I am running a tfjob from example main folder:
```➜  ks_app git:(master) ✗ kl mljob-kubeflow-t4r74-worker-0
Traceback (most recent call last):
  File "/app/main.py", line 90, in <module>
    fairing.config.set_model(TensorflowModel())
AttributeError: 'RuntimeConfig' object has no attribute 'set_model'```

Reply:
Ah set_model should be removed from runtime config as well. Sent a PR. You only need to call fairing.run, not set_model

https://github.com/kubeflow/fairing/pull/114

Reply:
k. I removed them as well.